ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -106,6 +106,11 @@ echo "export DS_OPENSHIFT_VERSION=\$(openshift_version)" >> /tmp/ds-vars.conf
 echo "export DS_REGISTRY=\$LOCAL_REGISTRY_DNS_NAME:\$LOCAL_REGISTRY_PORT" >> /tmp/ds-vars.conf
 echo "export DS_WORKING_DIR=\$WORKING_DIR" >> /tmp/ds-vars.conf
 echo "export DS_IP_STACK=\$IP_STACK" >> /tmp/ds-vars.conf
+
+echo "https://\$(oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > /tmp/console.url
```

Comment:
This block is meant to export just the configuration variables needed by the (next) `test` step. For the sake of clarity, it could be nice to have all the commands related to support the ci-chat-bot grouped together, and possibly executed only when a ci-chat-bot build is launched (and not for every job)

Reply:
Grouping is a good idea. I'll do that. But I don't think it is possible executing only ci-chat-bot, unfortunately there is no clue about this.

Reply:
Can't you use some of the default env vars exposed by ci-operator? Alternatively, you can either set your own directly in the job definitions (or, slightly more complex but probably more isolated, deploy a specific ci-chat-bot step).
What's the common approach for the other cluster profiles for exposing it?

Reply:
Since it looks like a no effect file for non cluster-bot runs, they all write it without checking with flag;
Most jobs are using this;
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L122
This is the other one specific to osd;
https://github.com/openshift/release/blob/master/ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh#L102

TBH, we might avoid adding additional flag for this purpose, we have already lots of :)